Huion_GC610/index.md

```diff
@@ -0,0 +1,20 @@
+---
+VID: "256c"
+PID: "006e"
+vendor: Huion
+product: GC610
+image: Huion_HGC610
```

Comment:
The reason the image is not displayed, is because the name here doesn't match the file name.